sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientTests.cs

```diff
@@ -599,6 +602,285 @@ public async Task HandlerPropertiesCanBeSetAfterEventProcessorHasStopped()
             Assert.That(() => processor.ProcessErrorAsync += eventArgs => Task.CompletedTask, Throws.Nothing);
         }
 
+        [Test]
+        public async Task StoppedClient_RelinquishesPartitionOwnerships_OtherClients_ConsiderThemClaimableImmediately()
```

Comment:
(extreme) nit:  If you wouldn't mind, would you remove the underscores for consistency with other test names?